client/src/components/settings/internet.tsx

```diff
@@ -142,15 +141,9 @@ class InternetSettings extends Component<InternetProps, InternetState> {
     if (!this.isFormPristine() && this.isFormValid()) {
       // // Only submit the form if is has changed, and if it is valid
       const { formValues } = this.state;
-      const isSocial = {
-        isGithub: !!formValues.githubProfile,
-        isLinkedIn: !!formValues.linkedin,
-        isTwitter: !!formValues.twitter,
-        isWebsite: !!formValues.website
-      };
 
-      const { updateInternetSettings } = this.props;
-      return updateInternetSettings({ ...isSocial, ...formValues });
+      const { updateSocials } = this.props;
+      return updateSocials({ ...formValues });
```

Comment:
```suggestion
      const isSocial = {
        isGithub: !!formValues.githubProfile,
        isLinkedIn: !!formValues.linkedin,
        isTwitter: !!formValues.twitter,
        isWebsite: !!formValues.website
      };
      return updateSocials({ ...isSocial, ...formValues });
```

Reply:
Yeah, I'd be in favour of removing those booleans. It doesn't make sense to have a boolean "github exists" value and the actual github link lol

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix